addons/mail/static/src/models/channel_member/channel_member.js

```diff
@@ -0,0 +1,79 @@
+/** @odoo-module **/
+
+import { registerNewModel } from '@mail/model/model_core';
+import { attr, many2one } from '@mail/model/model_field';
+import { insert, unlink, unlinkAll, insertAndReplace } from '@mail/model/model_field_command';
+
+function factory(dependencies) {
+
+    class ChannelMember extends dependencies['mail.model'] {
+
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+
+        /**
+         * @static
+         * @param {Object} data
+         * @returns {Object}
+         */
+        static convertData(data) {
+            const data2 = {};
+            if ('id' in data) {
+                data2.id = data.id;
+            }
+            // relations
+            if ('partner_id' in data) {
```

Comment:
If we can avoid this case it's better (always use the "partner" way which should return the result of mail partner format instead).